pallets/order-book/src/benchmarking.rs

```diff
@@ -33,6 +33,7 @@
 #![cfg(feature = "runtime-benchmarks")]
 // order-book
 #![cfg(feature = "wip")]
+// order-book
```

Comment:
there is a comment above 😃 
auto-formatter doesn't allow to have 1 line here.
I decided to move it before feature flag to not confuse anyone that the comment is related to `#![cfg(not(test))]` below.
btw it is minor, just decide and leave only one comment above or below

Reply:
I've added it on the same line to follow development process. Fmt moved it there :) Fixed